cli/src/commands/start.rs

```diff
@@ -710,7 +707,12 @@ impl Start {
         match node_type {
             NodeType::Validator => Node::new_validator(node_ip, self.bft, rest_ip, self.rest_rps, account, &trusted_peers, &trusted_validators, genesis, cdn, storage_mode, self.allow_external_peers, dev_txs, self.dev, shutdown.clone()).await,
             NodeType::Prover => Node::new_prover(node_ip, account, &trusted_peers, genesis, self.dev, shutdown.clone()).await,
-            NodeType::Client => Node::new_client(node_ip, rest_ip, self.rest_rps, account, &trusted_peers, genesis, cdn, storage_mode, self.rotate_external_peers, self.dev, shutdown).await,
+            NodeType::Client => {
+                if !self.nodisplay && !self.nocdn {
```

Comment:
Can you clarify why this is this only for Client and not Validator/Prover? 

Also, is the CDN syncing status still not being displayed?

Reply:
To show the CDN status, we would need to initialize the display earlier, similar to how it works with `snarkos-node-rest`. That would be a slightly bigger refactor and did not seem worth it to me right now.

Provers do not sync from CDN if I understand correctly. However, validators do, so let me change this.